state/migration_export.go

```diff
@@ -1120,13 +1119,6 @@ func (e *exporter) spaces() error {
 	e.logger.Debugf("read %d spaces", len(spaces))
 
 	for _, space := range spaces {
-		// We do not export the default space because it is created by default
```

Comment:
This works because we are defensive in migration_import, which skips exported default spaces. Otherwise we would get a uniqueness violation.

I think we need to keep this and instead, check for a "0" ID here:
https://github.com/juju/description/blob/master/model.go#L947

Reply:
done